packages/react/hover-card/src/HoverCard.stories.tsx

```diff
@@ -0,0 +1,696 @@
+                style={{ backgroundColor: 'purple' }}
+              >
+                <HoverCardArrow
+                  className={arrowClass}
+                  width={20}
+                  height={10}
+                  offset={20}
+                  style={{ fill: 'purple' }}
+                />
+                level 3
+              </HoverCardContent>
+            </HoverCard>
+          </HoverCardContent>
+        </HoverCard>
+
+        <HoverCardArrow
+          className={arrowClass}
+          width={20}
+          height={10}
+          offset={20}
+          style={{ fill: 'crimson' }}
+        />
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const CustomAnchor = () => {
+  return (
+    <HoverCard>
+      <HoverCardAnchor
+        style={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          width: 250,
+          padding: 20,
+          margin: 100,
+          backgroundColor: '#eee',
+        }}
+      >
+        Item
+        <HoverCardTrigger href="/" className={triggerClass}>
+          trigger
+        </HoverCardTrigger>
+      </HoverCardAnchor>
+      <HoverCardContent
+        className={contentClass}
+        side="right"
+        sideOffset={1}
+        align="start"
+        style={{ borderRadius: 0, width: 200, height: 100 }}
+      >
+        content
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const NonPortal = () => {
+  return (
+    <HoverCard>
+      <HoverCardTrigger href="/" className={triggerClass}>
+        trigger
+      </HoverCardTrigger>
+      <HoverCardContent portalled={false} className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        <CardContentPlaceholder />
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const WithSlottedTrigger = () => {
+  return (
+    <HoverCard>
+      <HoverCardTrigger as={Slot}>
+        <button className={triggerClass} onClick={() => console.log('StyledTrigger click')}>
+          trigger
+        </button>
+      </HoverCardTrigger>
+      <HoverCardContent className={contentClass} sideOffset={5}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        <CardContentPlaceholder />
+      </HoverCardContent>
+    </HoverCard>
+  );
+};
+
+export const WithSlottedContent = () => (
+  <HoverCard>
+    <HoverCardTrigger href="/" className={triggerClass}>
+      trigger
+    </HoverCardTrigger>
+    <HoverCardContent as={Slot} sideOffset={5}>
+      <div className={contentClass}>
+        <HoverCardArrow className={arrowClass} width={20} height={10} offset={10} />
+        <CardContentPlaceholder />
+      </div>
+    </HoverCardContent>
+  </HoverCard>
+);
+
+// change order slightly for more pleasing visual
+const SIDES = SIDE_OPTIONS.filter((side) => side !== 'bottom').concat(['bottom']);
+
+export const Chromatic = () => (
```

Comment:
Taken straight from the `Popover` visual tests, I felt like they covered this primitive equally well.